packages/dashboard/src/components/resizablePanes/index.tsx

```diff
@@ -67,11 +66,12 @@ export const ResizablePanes: FC<ResizablePanesProps> = ({ leftPane, centerPane,
 
     if (storedLeftWidthPercent) {
       const storedLeftWidth = elementWidth * storedLeftWidthPercent;
-      setLeftPaneWidth(storedLeftWidth);
+      setLeftPaneWidth(storedLeftWidth > DEFAULT_SIDE_PANE_WIDTH ? storedLeftWidth : DEFAULT_SIDE_PANE_WIDTH);
```

Comment:
DEFAULT_SIDE_PANE_WIDTH this should be DEFAULT_RESOURCE_EXPLORER_WIDTH ?

Reply:
but we are using same width as default for right (configuration panel) as well.